lib/Jsrt/ChakraCommon.h

```diff
@@ -2978,4 +2978,343 @@ typedef unsigned char* ChakraBytePtr;
         JsSetPromiseContinuationCallback(
             _In_ JsPromiseContinuationCallback promiseContinuationCallback,
             _In_opt_ void *callbackState);
+
+    /////////////////////
```

Comment:
TTD specific APIs should be in ChakraDebug.h or in ChakraCore.h, ChakraCommon.h contain runtime API which are exposed both in ChakraCore.dll and Chakra.dll. Also TTD is debugging specific so better to have it in ChakraDebug.h